rclpy/test/test_action_graph.py

```diff
@@ -51,6 +51,21 @@ def setUpClass(cls):
         cls.action_server20 = ActionServer(cls.node2, Fibonacci, TEST_ACTION0, lambda: None)
         cls.action_server21 = ActionServer(cls.node2, Fibonacci, TEST_ACTION1, lambda: None)
 
+        assert cls.wait_for_node(node=cls.node1, remote_node=cls.node0, timeout=2)
+        assert cls.wait_for_node(node=cls.node1, remote_node=cls.node2, timeout=2)
+        assert cls.wait_for_node(node=cls.node2, remote_node=cls.node0, timeout=2)
+        assert cls.wait_for_node(node=cls.node2, remote_node=cls.node1, timeout=2)
+
+    @staticmethod
+    def wait_for_node(*, node, remote_node, timeout=float('+inf')):
+        remote_node_identifier = (remote_node.get_name(), remote_node.get_namespace())
+        start = time.time()
+        while remote_node_identifier not in node.get_node_names_and_namespaces():
+            if time.time() - start > timeout:
+                return False
+            time.sleep(1)
```

Comment:
How about reducing this to something like `0.1` to speed up the test. Otherwise in case it case to wait it always waits a full second before retrying.

Reply:
Sounds good. Done in 1d9dc651a0fda1e7206af15e85c8f6a7973971ff.